[1 of 1] Compiling Data.Conduit.ImageSize ( Data/Conduit/ImageSize.hs, dist/build/Data/Conduit/ImageSize.o )

Data/Conduit/ImageSize.hs:92:11:
    Illegal polymorphic or qualified type:
      Consumer S.ByteString m (Maybe i)
    Perhaps you intended to use -XRankNTypes or -XRank2Types
    In the type signature for `getInt':
      getInt :: (Monad m, Integral i) =>
diff --git a/Data/Conduit/ImageSize.hs b/Data/Conduit/ImageSize.hs
index e0f09c8..4831adb 100644
--- a/Data/Conduit/ImageSize.hs
+++ b/Data/Conduit/ImageSize.hs
@@ -1,4 +1,4 @@
-{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE OverloadedStrings, Rank2Types #-}
 module Data.Conduit.ImageSize
     ( sinkImageSize
     , Size (..)
